google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java

```diff
@@ -83,6 +83,37 @@ private IntellijGoogleLoginService() {
     addLoginListenersFromExtensionPoints();
   }
 
+  @Nullable
+  private static Project getCurrentProject() {
+    Window activeWindow = WindowManagerEx.getInstanceEx().getMostRecentFocusedWindow();
+    if (activeWindow == null) {
+      return null;
```

Comment:
I'd rather we return Optional.empty() here and change the return to Optional<Project>?